example/burrow_integration/detail.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
I suggest to change licence into the same as in examples: https://github.com/hyperledger/iroha-python/blob/41a724b3f3f21921ad455d39fb959f45c98fe986/examples/batch-example.py#L4
```
Apache-2.0
```

Reply:
What difference will it make?

Reply:
> What difference will it make?

I'm not specialist, but as I know Apache is better than GPL, because when You use GPL code it makes Your code to be also GPL. Apache is not that sctict